src/Vogen/VoFilter.cs

```diff
@@ -68,7 +69,7 @@ public static bool IsTarget(INamedTypeSymbol? voClass)
         }
 
         AttributeData? voAttribute =
-            attributes.SingleOrDefault(a => a.AttributeClass?.FullName() is "Vogen.ValueObjectAttribute");
+            attributes.SingleOrDefault(a => a.AttributeClass?.FullName()?.StartsWith("Vogen.ValueObjectAttribute", StringComparison.Ordinal) == true);
```

Comment:
There's a similar method above that gets applicable attributes, so I've changed this to call that rather than do it itself.